src/lib/components/Error/PropValidator.tsx

```diff
@@ -44,5 +56,22 @@ export default function ErrorGenerator(swapWidgetProps: SwapWidgetProps) {
       }
     }
   }, [convenienceFee, convenienceFeeRecipient])
+
+  const { defaultInputAddress, defaultInputAmount, defaultOutputAddress, defaultOutputAmount } = swapWidgetProps
+  useEffect(() => {
+    if (defaultInputAmount && BigNumber.from(defaultInputAmount).lt(0)) {
```

Comment:
Is `lt` the same as `lessThan`? Use the more verbose method for readability.

Reply:
didn't know lessThan was a thing. cool.